src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/controllers/CreateReferralController.java

```diff
@@ -161,10 +170,16 @@ public ResponseEntity<CCDCallbackResponse> aboutToSubmitReferralDetails(
             )
         );
 
+        caseData.setReferredBy(String.format("%s %s", userDetails.getFirstName(), userDetails.getLastName()));
+
+        DocumentInfo documentInfo = createReferralService.generateCRDocument(caseData,
+            userToken, caseData.getEcmCaseType());
```

Comment:
caseData.getEcmCaseType - even though it's not being referenced later on, this should be the caseTypeId. CaseTypeId would return EnglandWales or Scotland whereas ecmCaseType is single or multiple.

Reply:
Pushed fix, thank you